src/Admin/Controllers/UsersController.cs

```diff
@@ -153,4 +163,10 @@ public async Task<IActionResult> Delete(Guid id)
 
         return RedirectToAction("Index");
     }
+
+    /* FEATURE FLAG CHECK, TO BE REMOVED IN THE FUTURE*/
+    public bool accountDeprovisioningEnabled()
```

Comment:
This method should be `private` as its not used anywhere else and use PascalCase for naming.

I would change this to return a nullable boolean like so:
```

public async Task<bool?> IsUserDomainVerifiedAsync(Guid userId)
    {
        return _featureService.IsEnabled(FeatureFlagKeys.AccountDeprovisioning)
            ? await _userService.IsManagedByAnyOrganizationAsync(userId)
            : null;
    }
```
With this approach, there's no need to pass the feature flag to the ViewModels. If the value is `null`, assume the feature is disabled and omit it from the view.